extensions/integration-tests/src/test/schemaCompare.test.ts

```diff
@@ -302,7 +302,7 @@ suite('Schema compare integration test suite @DacFx@', () => {
 		assert(includeResult.affectedDependencies[0].included === true, 'Table t2 should be included as a result of including v1. Actual: false');
 
 		// excluding views from the comparison should make it so t2 can be excluded
-		deploymentOptions.excludeObjectTypes.value.push(mssql.SchemaObjectType.Views);
+		deploymentOptions.excludeObjectTypes.value.push('Views');
```

Comment:
Can this instead use the existing enum and ToString it to make sure it stays linked?

Reply:
we have this defined in the deploymentOptions.ObjectTypes dictionary key as this. I think i can use the value of it and get the key to linked these! but the value can be hardcoded here!